dotCMS/src/main/java/com/dotcms/system/event/local/type/pushpublish/PushPublishStartEvent.java

```diff
@@ -1,17 +1,22 @@
 package com.dotcms.system.event.local.type.pushpublish;
 
+import com.dotcms.publisher.business.PublishQueueElement;
+import com.dotcms.system.event.local.type.publish.PublishEvent;
+
+import java.util.Date;
+import java.util.List;
+
 /**
  * Object used to represent an event to be triggered when the push publishing process starts
  *
  * @author nollymar
  */
-public class PushPublishStartEvent extends PushPublishEvent {
+public class PushPublishStartEvent extends PublishEvent {
 
-    public PushPublishStartEvent(String name) {
-        setName(name);
+    public PushPublishStartEvent(List<PublishQueueElement> publishQueueElements) {
+        this.setName(PushPublishStartEvent.class.getCanonicalName());
+        this.setPublishQueueElements(publishQueueElements);
+        this.setDate(new Date());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use the Java 8 Date and Time API instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2143)